bin/check-cloudwatch-metric.rb

```diff
@@ -91,11 +91,11 @@ class CloudWatchMetricCheck < Sensu::Plugin::Check::CLI
   option :compare,
          description: 'Comparision operator for threshold: equal, not, greater, less',
          short: '-o OPERATION',
-         long: '--opertor OPERATION',
+         long: '--operator OPERATION',
```

Comment:
while I agree with this, this unfortunately is a breaking change meaning we need to do a major version release.